### ISSUE TITLE
Unable to use store.get to get nested variable with dotpath if path includes arrays

### ISSUE THREAD

**Issue by hybridwebdev** [davestewart/vuex-pathify#34]
So as the title explains, is that unless I am missing something, it's not possible to use store.get() with a dot-path like this `some.nested.prop.0.apples` given a structure like this: 
` { some: { nested: { prop: [{apples: 'juicy'}] } } } `
I also tried `some.nested.prop[0].apples` which also doesn't work. However `some.nested.prop` will give you an array which of course contains the apples object. However, this is undesirable as this means you have to know the structure of the data ahead of time. 

Hoping you can shed a light on this. Ps, I am just mind BLOWN over how amazing this project is. Keep up the great work, and thanks for such an awesome tool. It took me a while to wrap my head around how it all works, and honestly the @ decorator is a bit...err ... annoying to work around, however I read your bit about why you opted to use it which makes sense.

**Comment by davestewart** [comment]
Hey, hey,

Sorry for the confusion.

I decided to not support arrays on purpose, as the original intention was to support concrete properties, and I considered that the fluid nature of arrays would only add confusion:

https://davestewart.github.io/vuex-pathify/#/api/paths?id=sub-property-access

But you make a valid use case for using `store.get()`. I can't promise to add this in straight away, but I am persuaded to take a look at it, especially as the lib has gained some traction now and seems like it's pretty stable for everyone!

As for using `.` vs `@`, I am considering this.

I could potentially just use [lodash](https://lodash.com/docs/4.17.10#get) to support universal paths, but I do need to review how the lib will determine the mutation boundary.

**Comment by hybridwebdev** [comment]
Well dang, I didn't expect such a quick reply. I actually came back to edit and mention I had found this: https://github.com/davestewart/vuex-pathify/issues/19 which explains why this doesn't work. I'll consider this issue closed, but it would be great to see this added. I can personally say that lodash handles the aforementioned dot path structure (including array's) just fine, as that's what I'm using as a solution now.
